internal/services/rewards_test.go

```diff
@@ -806,6 +806,139 @@ func TestBaselineIssuance(t *testing.T) {
 	}
 }
 
+func TestSyntheticDeviceIssuance(t *testing.T) {
+	ctx := context.Background()
+
+	settings, err := shared.LoadConfig[config.Settings]("settings.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	settings.TransferBatchSize = 2
+
+	logger := zerolog.Nop()
+
+	cont, conn := utils.GetDbConnection(ctx, t, logger)
+	defer func() {
+		err := cont.Terminate(ctx)
+		assert.NoError(t, err)
+	}()
+
+	scens := []Scenario{
+		{
+			Name: "AutoPiJoinLevel1",
```

Comment:
AutoPi?

Reply:
It seems like we're mostly checking that things don't explode here, which is good but maybe not at the level of certainty that we'd prefer. Let me think about how to do this—the tests here are gross, I grant that.